### DISCUSSION TITLE
[ASK] Have you considered joining the LXQT project?

### DISCUSSION THREAD

**Discussion by theavege** [doublecmd/doublecmd#1971] [General]
Have you considered joining the [LXQT](https://lxqt-project.org/about/) project?

**Comment by alexx2000** [comment]
DC is a cross-platform file manager. DC does not depend on any desktop environment.

**Reply by theavege** [reply]
I know that. I use your program myself because I really like the implementation of the copy queue.
I think that joining Lazarus programs to the LXQT project will be mutually beneficial

Thank you for answer.
